ultravox/model/ultravox_model.py

```diff
@@ -75,6 +78,10 @@ def get_decoder(self):
     def tie_weights(self):
         return self.language_model.tie_weights()
 
+    def set_loss_config(self, loss_config: Optional[LossConfig] = None):
```

Comment:
Why `Optional`?

Reply:
This is because `loss_config` defaults to None in `config_base.py`, but it shouldn't be None during training. I will update `config_base.py` to make `loss_confg` to have a non-None default value, then we can get rid of Optional here.

Reply:
It turns out that I have to use Optional[LossConfig] in config_base.py, otherwise it will not pass `just check`.  I updated `train.py` to only invoke `set_loss_config` when `args.loss_config is not None`